productcomments.php

```diff
@@ -164,13 +164,34 @@ public function getCacheId($id_product = null)
     protected function _postProcess()
     {
         if (Tools::isSubmit('submitModerate')) {
-            Configuration::updateValue('PRODUCT_COMMENTS_MODERATE', (int) Tools::getValue('PRODUCT_COMMENTS_MODERATE'));
-            Configuration::updateValue('PRODUCT_COMMENTS_ALLOW_GUESTS', (int) Tools::getValue('PRODUCT_COMMENTS_ALLOW_GUESTS'));
-            Configuration::updateValue('PRODUCT_COMMENTS_USEFULNESS', (int) Tools::getValue('PRODUCT_COMMENTS_USEFULNESS'));
-            Configuration::updateValue('PRODUCT_COMMENTS_COMMENTS_PER_PAGE', (int) Tools::getValue('PRODUCT_COMMENTS_COMMENTS_PER_PAGE'));
-            Configuration::updateValue('PRODUCT_COMMENTS_ANONYMISATION', (int) Tools::getValue('PRODUCT_COMMENTS_ANONYMISATION'));
-            Configuration::updateValue('PRODUCT_COMMENTS_MINIMAL_TIME', (int) Tools::getValue('PRODUCT_COMMENTS_MINIMAL_TIME'));
-            $this->_html .= '<div class="conf confirm alert alert-success">' . $this->trans('Settings updated', [], 'Modules.Productcomments.Admin') . '</div>';
+            $errors = [];
+            $pcmt = Tools::getValue('PRODUCT_COMMENTS_MINIMAL_TIME');
+            if (!Validate::isUnsignedId($pcmt)) {
```

Comment:
Why using isUnsignedId on a integer value that is not an id ?
```suggestion
            if (!Validate::isUnsignedInt($productCommentsMinimalTime)) {
```